decidim-participatory_processes/app/cells/decidim/participatory_processes/content_blocks/highlighted_processes/show.erb

```diff
@@ -1,14 +1,13 @@
-<section id="highlighted_processes" class="wrapper-home home-section">
-  <div class="row" id="highlighted-processes">
-    <h3 class="section-heading"><%= t("active_processes", scope: i18n_scope) %></h3>
-    <div class="row collapse">
-      <div class="row small-up-1 smallmedium-up-2 mediumlarge-up-3
-        large-up-4 card-grid">
-        <% highlighted_items.each do |item| %>
-          <%= card_for item, size: :g %>
-        <% end %>
-      </div>
-    </div>
+<section id="highlighted-processes" class="home__section">
+  <div class="home__section-header">
```

Comment:
in this particular template, we cannot see `<section id "highlighted_processes"` , followed by a `<div id="highlighted-processed.">`. Maybe this is the way it should work. and it would be in any other templates commented with "
Do we really need to have a section id= and the next element div id ?"